great_expectations/rule_based_profiler/data_assistant/onboarding_data_assistant.py

```diff
@@ -316,19 +310,6 @@ def _build_numeric_columns_low_precision_rule() -> Rule:
             metric_value_kwargs=None,
             json_serialize=True,
         )
-        column_quantile_values_range_parameter_builder_for_validations: ParameterBuilder = DataAssistant.commonly_used_parameter_builders.build_numeric_metric_range_multi_batch_parameter_builder(
-            metric_name="column.quantile_values",
-            metric_value_kwargs={
-                "quantiles": f"{VARIABLES_KEY}quantiles",
-                "allow_relative_error": f"{VARIABLES_KEY}allow_relative_error",
-            },
-            json_serialize=True,
-        )
-        column_median_values_range_parameter_builder_for_validations: ParameterBuilder = DataAssistant.commonly_used_parameter_builders.build_numeric_metric_range_multi_batch_parameter_builder(
```

Comment:
Same question - what makes median different from min and max?

Reply:
Because min and max are extreme values, they should not need more precision.